RunningOrder/Story/Models/Link.swift

```diff
@@ -8,15 +8,23 @@
 
 import Foundation
 
-struct Link {
+struct Link: Identifiable {
+    var id = UUID()
```

Comment:
As it was before LinkEntity, I didn't see these changes :
why did you add uuid ? please remove it, we don't need it.

please add the conformance to identifiable in an extension, as other conformance. the id needed for identifiable could be something simpler such as the label. (does it have sense to have multiple links with the same label ?)